### DISCUSSION TITLE
Import of Csv files with "Import-DbaCsv" generate Invalid cast from 'System.String' to 'System.Guid'

### DISCUSSION THREAD

**Discussion by Johnn012** [dataplat/dbatools#7063] [Q&A]
I am new in using dbatools and PowerShell :)
I want to import a csv file containing GUID into my SQL database (see the code below), but i get the error "Invalid cast from 'System.String' to 'System.Guid'". 

does anyone have an idea how I can solve this problem?
Thanks in advance!

Import-DbaCsv -Path $Dir -SqlInstance $SQLServer -Database $DatabaseName -Schema $Shema -Table $TableName -AutoCreateTable -SingleColumn -KeepIdentity -Encoding UTF8  -Delimiter "," -Quote '"' -UseColumnDefault -Verbose

**Comment by GrabASalad** [comment]
the only way i have found to change the type of the destination column to varchar(40) but that isnt quick and dirty it's more quick and filthy...
